dazel/lib/src/bazelify/build.dart

```diff
@@ -604,3 +590,32 @@ String _sourcesToGlob(
         Iterable<String> sources, Iterable<String> excludeSources) =>
     'glob([${sources.map((s) => '"$s"').join(", ")}], '
     'exclude=[${excludeSources.map((s) => '"$s"').join(", ")}])';
+
+/// Checks an html [file] to see if it is a dart app. If so then returns a
```

Comment:
Can we combine them into 1 sentence? Or find another sentence that can serve as the title?
https://www.dartlang.org/guides/language/effective-dart/documentation#do-make-the-first-sentence-a-standalone-paragraph